### ISSUE TITLE
Ability to add & edit alt-text for uploaded images

### ISSUE THREAD

**Issue by caaster** [sul-dlss/exhibits#2462]
This is a multi-faceted a11y requirement. As an exhibit creator, I would expect the following behavior:

- When I upload a single non-SDR image to my exhibit (Curation > Items > Add items > Upload item), there should be a field automatically provided for me to add alt-text. This is the alt-text that I would expect to be available to screen readers on the Item details page (OpenSeaDragon viewer).
- When I bulk upload non-SDR images to my exhibit (Curation > Items > Add items > Upload multiple items), there should be a field automatically provided for me to add alt-text for each image. This is the alt-text that I would expect to be available to screen readers on the Item details page (OpenSeaDragon viewer).
- When I use a widget to display an image I have uploaded using the Upload item or the Upload multiple items options, I need to be able to edit the alt text on each page to accurately reflect the context on each page. The widgets that might be used include: Item row, Item carousel, Item embed, Item slideshow, and possibly Item grid. I may need to make a different alt-text edit to each page where the image is displayed so that the appropriate context can be provided.
- When I use a widget to display an image that I have uploaded using the Item row widget, I need the ability to add alt-text for that image. 
- Further investigation needed for the iframe and video widgets, where the ability to add alt-text may also be needed.

**Comment by corylown** [comment]
Some context from an older related issue on this topic: https://github.com/sul-dlss/exhibits/issues/819
